lib/arel/visitors/spanner.rb

```diff
@@ -22,6 +33,29 @@ def bind_block
       end
 
       # rubocop:disable Naming/MethodName
+      def visit_Arel_Nodes_OptimizerHints o, collector
+        o.expr.each do |v|
+          if v.start_with? "max_staleness:"
+            collector.hints[:staleness] = \
+              StalenessHint.new max_staleness: v.delete_prefix("max_staleness:").to_f
```

Comment:
Nits: can we call `next` here? Same for the below if blocks. I think they are exclusive to each other, right?

Reply:
Yes. Done.